com.unity.ml-agents/Runtime/Policy/BehaviorParameters.cs

```diff
@@ -44,16 +47,26 @@ enum BehaviorType
         [Tooltip("Use all Sensor components attached to child GameObjects of this Agent.")]
         bool m_UseChildSensors = true;
 
+        /// <summary>
+        /// The associated <see cref="BrainParameters"/> for this behavior.
```

Comment:
How does `see cref` get expanded? Seems like either this or the one for `behaviorName` would look bad.

Reply:
If `<see cref="Foo"/>` results in "Foo" in the docs, then the `behaviorName` doc would look like

> The name of this behavior, which is used as a base name.
> fullyQualifiedBehaviorName

If not, then this will look like

> The associated for this behavior.

Reply:
It gets expanded and linked. I fixed `fullyQualifiedBehaviorName`. In doxygen, it now looks like: 
![docs](https://user-images.githubusercontent.com/477636/75308158-8a330600-5802-11ea-94f1-d8f8d2f45532.png)

Reply:
Cool, thanks for clarifying